R/render_email.R

```diff
@@ -0,0 +1,72 @@
+#' R Markdown render functions for the `blastula_email` output format
+#'
+#' The `render_email()` and `render_connect_email()` functions both allow for
+#' rendering an an email message. We can supply an R Markdown document (.Rmd)
+#' with the output specified as `output: blastula::blastula_email`. While the
+#' `render_email()` and `render_connect_email()` functions have similar
+#' arguments, the `render_connect_email()` is preferred when publishing to the
+#' RStudio Connect service. It allows for the inclusion of a predefined footer
+#' that contains useful links for email recipients.
+#'
+#' @param input The input file to be rendered. This should be an R Markdown
+#'   document (.Rmd) with the output specified as `output:
+#'   blastula::blastula_email`.
+#' @param envir The environment in which the code chunks are to be evaluated
+#'   during knitting (we can use `new.env()` to guarantee an empty, new
+#'   environment).
+#' @param quiet An option to suppress printing of the Pandoc command line
```

Comment:
The rmarkdown docs are misleading here. This should actually be "to suppress printing of the Pandoc command's output", I think.

Reply:
I changed this to: `An option to suppress printing of the command line output from Pandoc during rendering`. It seems more accurate and I hope it's not a clunky sentence.